specification/columns/chargeclassification.md

```diff
@@ -0,0 +1,38 @@
+# Charge Classification
+
+A Charge Classification indicates whether the row represents a regular cost record or a correction record. Charge Classification is a supplemental detail to the [Charge Category](#chargecategory). It provides additional context to describe the nature of the charge.

```

Comment:
I don't love this verbiage because, ultimately, it has no relationship to Charge Category.  Yes, we've talked ad nauseum about this column in the context of Charge Category -- but they have a many-to-many relationship.  I believe we can skip any reference to Charge Category and simply say something like:

```suggestion
A Charge Classification indicates whether the row represents a regular billed line item or a correction to one or more previously billed line items.

```